.gitignore

```
@@ -5,6 +5,9 @@ _site/
 .jekyll-cache/
 .jekyll-metadata
 
+# Pycharm project
+/.idea
```

The `/` shouldn't be here, right?
```suggestion
.idea
```